src/modules/launcher/Wox.Plugin/ThemeManager.cs

```diff
@@ -132,7 +139,7 @@ private void ChangeTheme(Theme theme)
             {
                 currentTheme = Theme.None;
             }
-            Debug.WriteLine("Theme Changed to :" + currentTheme);
```

Comment:
Can we have a log instead?

Reply:
I tried that but it results in a circular dependency. Logger is part of `Wox.Infrastructure` project which has dependency on `Wox.Plugins`. For accessing logger in `Wox.Plugins`, we would need to add dependency on `Wox.Infrastructure` resulting in a circular dependency. The solution here would be to move `PublicAPIInstance` to `Wox.Infrastructure` but doesn't seem logically correct location for this class.

Reply:
Was there any reason behind moving it from the `PowerLauncher` project to the `Wox.Plugin` project?

Reply:
Yes! `PublicAPIInstance` is in the namespace of Wox.Plugin and it would cause a circular dependency issue if we try to access `ThemeManager` in `PublicAPIInstance`.